docs/folders.md

```diff
@@ -45,9 +49,13 @@ This is where styling for the app is housed. If colors, fonts, spacing, etc. nee
 
 This folder contains any static files that are used by the app, such as images.
 
+#### `components/`
+
+This folder contains the components referenced in `App.js`.
+
 #### `config/`
 
-In the `config/` directory, there are `.js` files which contain settings for the different parts of the task.  Every task should have a `main` config and a `trigger` config (assuming use of the event marker). The `main` config has all global settings for the task (such as whether it is in mturk mode or not), load the appropriate language file, and set up a default (or only) configuration object for the task. The `trigger` config has settings specific to the event marker and uses a slightly different style of javascript as it is imported both in the React app as well as the electron process.
+In the `config/` directory, there are `.js` files which contain settings for the different parts of the task.  Every task should have a `main` config and a `trigger` config (assuming use of the event marker). The `main` config has all global settings for the task (such as whether it is in mturk mode or not), load the appropriate language file, and set up a default (or only) configuration object for the task. Different default configurations can be loaded into .env files in the `env` directory. The `trigger` config has settings specific to the event marker and uses a slightly different style of javascript as it is imported both in the React app as well as the electron process.
```

Comment:
specify these configurations are for features as opposed to in-task config.  the config in the `main` is also many other types of config and we should differentiate these.  Maybe a short example might be useful `(e.g. the `.env.clinic` file enables event marker, photodiode, and .... )`